src/ert/analysis/_es_update.py

```diff
@@ -168,18 +168,18 @@ def _load_observations_and_responses(
     ],
 ]:
     try:
-        observed_responses_data = ensemble.get_measured_data(
+        observations_and_responses = ensemble.get_observations_and_responses(
             [*selected_observations], iens_active_index
         )
     except KeyError as e:
         # Exit early if some observations are pointing to non-existing responses
         raise ErtAnalysisError("No active observations for update step") from e
 
-    S = observed_responses_data.vec_of_realization_values()
-    observations = observed_responses_data.vec_of_obs_values()
-    errors = observed_responses_data.vec_of_errors()
-    obs_keys = observed_responses_data.vec_of_obs_names()
-    indexes = observed_responses_data.vec_of_obs_indexes()
+    responses = observations_and_responses.responses()
+    observations = observations_and_responses.observations()
+    errors = observations_and_responses.errors()
+    observation_keys = observations_and_responses.observation_keys()
+    indexes = observations_and_responses.observation_indexes()
```

Comment:
I wonder if this `observations_and_responses.observation_indexes()` (I'm the one who called it observation_index in the first place), could be called something else? Because this refers to the primary key of the response. For example for gen_data it is `["index", "report_step"]`, while for summary it is `time`, how about just `index`?